core/src/test/java/nl/nn/adapterframework/jta/btm/BtmDataSourceFactoryTest.java

```diff
@@ -32,5 +33,7 @@ public void testSetup() {
 		assertEquals(2, factory.getMaxPoolSize());
 		assertEquals(3, factory.getMaxIdleTime());
 		assertEquals(4, factory.getMaxLifeTime());
+
+		assertNull(factory.getTestQuery());
```

Comment:
Kunnen we de test niet aanpassen dat wordt getest dat er wél iets in de test-query staat?

Reply:
Ja dat kan, mijn doel was om te kijken of hij default uit stond. Dit is eigenlijk een fallback voor wanneer `Connection.isValid()` (JDBC4) niet is geimplementeerd.

Reply:
Dan houden we het zo.